schemas/oic.r.sp.json

```diff
@@ -0,0 +1,34 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description" : "Copyright (c) 2017 Open Connectivity Foundation, Inc. All rights reserved.",
+  "id": "https://www.openconnectivity.org/ocf-apis/security/schemas/oic.r.sp.json#",
+  "definitions": {
+    "oic.r.sp": {
+      "type": "object",
+      "properties": {
+        "supported_profiles": {
+          "type": "array",
+          "description": "Array of supported security profiles",
+          "items": {
+            "$ref": "oic.sec.sp.name.json#/definitions/oic.sec.sp.name"
+          }
+        },
+        "active_profile": {
+          "description": "Security profile currently active",
+          "$ref": "oic.sec.sp.name.json#/definitions/oic.sec.sp.name"
+        },
+        "credid":  {
+          "type": "integer",
+          "description": "A value is returned if the security profile defines its usage. When defined, credid Property refers to the /oic/sec/cred Resource entry where the credential is used to authorize the available security profiles (e.g. OCF Rooted Certificate Chain)",
+          "readOnly": true
+        }
+      }
+    }
+  },
+  "type": "object",
+  "allOf": [
+    { "$ref": "../../core/schemas/oic.core-schema.json#/definitions/oic.core" },
```

Comment:
Wouter can confirm but I don't think we can use ".." in our references.  It breaks the tooling...